File: 168-excel-sheet-column-title/excel-sheet-column-title.ts
function convertToTitle(columnNumber: number): string {

    let str = '';
    while(columnNumber!=0){
        columnNumber--;
        let alpa = columnNumber%26;
        str = String.fromCharCode(65+alpa) + str
        columnNumber = Math.floor(columnNumber/26);
    }

    return str;
    
};

File: 168-excel-sheet-column-title/excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int columnNumber) {

    StringBuilder str = new StringBuilder();
    while(columnNumber!=0){
        columnNumber--;
        int alpa = columnNumber%26;
        str.append((char) (alpa + 'A'));
        columnNumber = columnNumber/26;
    }

    return str.reverse().toString();
        
    }
}